core/server/src/test/java/alluxio/worker/block/BlockWorkerTest.java

```diff
@@ -106,10 +108,11 @@ public void before() throws IOException {
   }
 
   /**
-   * Resets the worker context.
+   * Resets the worker context and close the data server on clean up.
```

Comment:
This comment is the line which should have been changed to just reset the worker context.